Makefile

```
@@ -138,9 +138,11 @@ vendor-bin/behat/vendor: vendor/bamarni/composer-bin-plugin vendor-bin/behat/com
 
 vendor-bin/behat/composer.lock: vendor-bin/behat/composer.json
 	@echo behat composer.lock is not up to date.
+	composer bin behat update --no-progress --lock
 
 composer.lock: composer.json
 	@echo composer.lock is not up to date.
+	composer update --no-progress --lock
```

looks like it doesn't work for other make targets. we can do
```suggestion
	@rm vendor-bin/behat/composer.lock || true

composer.lock: composer.json
	@echo composer.lock is not up to date.
	@rm composer.lock || true
```